deploy/azuredeploy.json

```diff
@@ -67,7 +67,7 @@
           },
           "properties": {
             "ResourceConnectionString": "[listkeys(variables('commsName'), '2020-08-20-preview' ).primaryConnectionString]",
-            "WEBSITE_NODE_DEFAULT_VERSION": "12.13.0"
+            "WEBSITE_NODE_DEFAULT_VERSION": "~12",
```

Comment:
node version 12 is [soon to be deprecated](https://nodejs.org/en/about/releases/), can we specify something like 12 or greater?

Reply:
While 16 is the latest LTS we do have customers that are using much older versions of node. Does it not make sense to just require the minimum version in order to run the sample?

Reply:
Good point, yeah that's a good idea lets keep it v12.

Reply:
I think there is value in allowing the min version of node to make sure this runs. We have customers that are always pushing at our min NPM limit. If we force people to always be on the latest LTS, we are potentially missing out on some of our customers. Curious if any others have any suggestions here.